VERSION.cmake

```diff
@@ -1,6 +1,6 @@
 SET( VERSION_MAJOR "4")
 SET( VERSION_MINOR "2" )
-SET( VERSION_PATCH "6" )
+SET( VERSION_PATCH "7" )
```

Comment:
Do we need the version bump as it does not change the package at all? For users there is no change (besides the docu available online).

Reply:
I don't know, but I learned over the years that _some_ tool will always complain if we don't do a version bump and a change log entry with a bug number...

Reply:
The Jenkins job submits the change *only if the version is changed*. If we do not bump the version no SR is done and thus no bugzilla number is required. It will be sumbitted together with the next change (which will very likely include a bug number).

IMHO the version bump is not need. But on the other hand it won't harm...

Reply:
Right, it won't harm; it's not as if we are running out of numbers. ;-)
And the corresponding change log entry gives users some information what we did.